### ISSUE TITLE
install_requires in 0.14.0 is incorrect

### ISSUE THREAD

**Issue by mgorny** [shlomif/pysol_cards#4]
```
      install_requires=['# The order of packages is significant, because pip processes them in the order','# of appearance. Changing the order has an impact on the overall integration','# process, which may cause wedges in the gate later.','coverage','pytest','pytest-cov','random2','requests','six','twine'],
```

This contains test and development requirements that are almost certainly unnecessary on the installed package.

**Comment by mgorny** [comment]
Also, I'm not sure if comments are even allowed there, like this.

**Comment by shlomif** [comment]
@mgorny : hi! Thanks for the report. I avoided the comments' lines but 'pytest'/etc. are added by https://github.com/Kwpolska/python-project-template (though I can try to remove them) and I think they do not hurt much.

**Comment by SoapGentoo** [comment]
@shlomif they hurt, because `setuptools` verifies that they are installed when importing the package in python. This means that `pytest` needs to be installed for the package to be usable, which is not the case for 95% of Gentoo users. Please remove it and consider adding it to `extras_require` or some other field that doesn't require all users to have pytest installed.

**Comment by shlomif** [comment]
@SoapGentoo @mgorny : hopefully fixed  in 0.14.2 . Please consider reciprocating: https://www.shlomifish.org/meta/how-to-help/ .

**Comment by shlomif** [comment]
Closing due tolack of reply/
